### PR TITLE
virsh_dump: Add document test case

### PR DESCRIPTION
**Pull request by cliping**
Signed-off-by: Liping Cheng <lcheng@redhat.com>

### CODE REVIEW

**Comment by cliping** [comment]
Test result:

#avocado run --vt-type libvirt --vt-machine-type q35 virsh.dump.doc_test --vt-connect-uri qemu:///system
/root/.libvirt-ci-venv-ci-runtest-wk3BTh/lib/python3.6/site-packages/avocado_framework-80.0-py3.6.egg/avocado/plugins/run.py:292: FutureWarning: The following arguments will be changed to boolean soon: sysinfo, output-check, failfast and keep-tmp. 
  FutureWarning)
JOB ID     : b606ebe686885fcdb2bb90271536c4ca3d1d85b9
JOB LOG    : /root/avocado/job-results/job-2020-07-26T23.26-b606ebe/job.log
 (1/1) type_specific.io-github-autotest-libvirt.virsh.dump.doc_test: PASS (34.83 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 37.40 s

File: libvirt/tests/src/virsh_cmd/domain/virsh_dump.py [hunk 1]
```diff
@@ -217,6 +218,18 @@ def check_dump_format(dump_image_format, dump_file):
...
+    # check the explanation of "--memory-only" option in virsh dump man.
+    if document_string:
+        logging.info("document string: %s" % document_string)
+        ret = process.run("man virsh", shell=True)
+        if ret.exit_status:
+            test.error("failed to run 'man virsh'.")
+        man_virsh = ret.stdout_text.strip()
+        for item in document_string:
+            if item not in man_virsh:
+                test.fail("failed to check document string in virsh man page.")
```
Comment (chloerh) [inline libvirt/tests/src/virsh_cmd/domain/virsh_dump.py:230]: I think we could use all() here
```
if not all(item in man_virsh for item in docxxxx]):
    test.fail(xxxx....
```

File: libvirt/tests/src/virsh_cmd/domain/virsh_dump.py [hunk 2]
```diff
@@ -217,6 +218,18 @@ def check_dump_format(dump_image_format, dump_file):
...
+        logging.info("the document string in virsh man page.")
+        return 0
```
Comment (chloerh) [inline libvirt/tests/src/virsh_cmd/domain/virsh_dump.py:232]: And it seems just 'return' would be enough, right?

File: libvirt/tests/src/virsh_cmd/domain/virsh_dump.py (hunk 1 above)
Comment (cliping) [reply libvirt/tests/src/virsh_cmd/domain/virsh_dump.py]: Updated

File: libvirt/tests/src/virsh_cmd/domain/virsh_dump.py (hunk 2 above)
Comment (cliping) [reply libvirt/tests/src/virsh_cmd/domain/virsh_dump.py]: Updated

**Pull request merged**